src/snowflake/snowpark/functions.py

```diff
@@ -6702,17 +6702,75 @@ def sproc(
         registered. Invoking :func:`sproc` with ``replace`` set to ``True`` will overwrite the
         previously registered function.
 
+        4. To describe the return type for a table stored procedure, use
```

Comment:
Maybe change 
"To describe the return type for a table stored procedure" 
to 
"To describe the return type for a stored procedure that returns tabular data"